<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_quiz_window</class>
 <widget class="QDialog" name="create_quiz_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Quiz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: #f0f0f3;
    border-radius: 20px;
}
QFrame#frame {
    background-color: #e0e0e0;
    border-radius: 15px;
    border: 1px solid #b0b0b0;
}
QLabel {
    background-color: #f5c211;
    border: 1px solid #b0b0b0;
    border-radius: 10px;
    font-size: 16px;
}
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #b0b0b0;
    border-radius: 10px;
    padding: 8px;
    font-size: 14px;
}
QPushButton#confirm_button {
    background-color: #4caf50;
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
QPushButton#confirm_button:hover {
    background-color: #45a049;
}
QPushButton#back_to_main_button {
    background-color: #2196f3;
    color: white;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
}
QPushButton#back_to_main_button:hover {
    background-color: #1e88e5;
}
   </string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>520</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>                 Here you can create your own quiz, good luck!</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_quiz_theme_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter quiz theme:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theme_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Theme</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_difficulty_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose difficulty:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choose_difficulty_lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1, 2 or 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="ammount_of_points_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of points:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="points_lineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Points</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_question_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter question:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="question_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QLabel" name="enter_answer_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>215</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter answers:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_A_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>245</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Answer A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_B_lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>245</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Answer B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_C_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>295</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Answer C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_D_lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>295</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Answer D</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_correct_answer_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>215</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose correct answer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choose_correct_answer_lineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>245</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>A, B, C or D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_to_main_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to main menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
